client/src/js/modules/dc/views/reprocess2.js

```diff
@@ -156,8 +156,8 @@ define(['backbone', 'marionette', 'views/dialog',
             if (e) e.preventDefault()
 
             if (this.aps.length) {
-                var e = this.aps.at(0)
-                var c = e.get('CELL')
+                var a = this.aps.at(0)
+                var c = a.get('CELL')
```

Comment:
Might be a good opportunity to switch these out for `let` or `const` instead, which does not allow in-scope redeclaration of variables

```suggestion
                const a = this.aps.at(0)
                const c = a.get('CELL')
```